frontend/test/e2e/caselaw/management-data/duplicate-check.spec.ts

```diff
@@ -0,0 +1,415 @@
+          page,
+          prefilledDocumentUnit,
+        }) => {
+          await navigateToSearch(page)
+          await test.step("Enter core data into search with different court", async () => {
+            await page
+              .getByLabel("Aktenzeichen Suche")
+              .fill(prefilledDocumentUnit.coreData.fileNumbers?.[0] ?? "")
+            const date = DateUtil.formatDate(
+              prefilledDocumentUnit.coreData.decisionDate,
+            )
+            await page
+              .getByLabel("Entscheidungsdatum Suche")
+              .first()
+              .fill(date!)
+            await page.getByLabel("Gerichtstyp Suche").fill("BGH")
+          })
+
+          await test.step("Search and create doc unit from empty search results", async () => {
+            await page.getByLabel("Nach Dokumentationseinheiten suchen").click()
+            await page
+              .getByRole("button", { name: "Übernehmen und fortfahren" })
+              .click()
+          })
+
+          await expectDuplicateWarning(page)
+
+          const docNumber = await page
+            .getByTestId("document-unit-info-panel")
+            .getByRole("heading")
+            .textContent()
+
+          // If the test fails, the doc unit will not be deleted.
+          await deleteDocumentUnit(page, docNumber!)
+        })
+      },
+    )
+
+    test("When duplicate is ignored, no warning is shown", async ({
+      page,
+      documentNumber,
+      prefilledDocumentUnit,
+    }) => {
+      await test.step("Create duplicate by editing core data", async () => {
+        await navigateToCategories(page, documentNumber)
+        await setFileNumberToMatchDocUnit(page, prefilledDocumentUnit)
+        await setDocTypeToMatchDocUnit(page, prefilledDocumentUnit)
+        await save(page)
+        await expectDuplicateWarning(page)
+      })
+
+      await test.step("Check management data and ignore warning", async () => {
+        await navigateToManagementData(page, documentNumber)
+        await expectDocUnitSummaryInDuplicatesList(page, prefilledDocumentUnit)
+        await ignoreDuplicateWarning(page, prefilledDocumentUnit)
+        await expectNoDuplicateWarning(page)
+
+        await test.step("Duplicate warning should be ignored on other doc unit as well", async () => {
+          await expect(page.getByLabel("Warnung ignorieren")).toBeChecked()
+        })
+      })
+
+      await test.step("Check and reset warning on other doc unit", async () => {
+        const prefilledDocNumber = prefilledDocumentUnit.documentNumber
+        await navigateToManagementData(page, prefilledDocNumber)
+        await expectNoDuplicateWarning(page)
+        await test.step("Reset duplicate warning", async () => {
+          await page.getByLabel("Warnung ignorieren").uncheck()
+        })
+        await expectDuplicateWarning(page)
+      })
+    })
+
+    // FIXME: ignoring second duplicate warning is flaky
```

Comment:
TODO: Test is flaky

Reply:
Locally it's of course not flaky 😢 but when running on bulk against staging, it seems to be 20% failure rate. :-/